### PR TITLE
[cms] fix: fetch invoice proposals lineitem correctly

### LINKED ISSUE #2162: TypeError: h is not a function
I hestitated to post this because I am not able to reproduce it reliably. I opened a bunch of other users invoices in new tabs, and on a couple of them I got this.  Went away after refreshing.

```
TypeError: h is not a function
    at Detail.jsx:58
    at is (react-dom.production.min.js:211)
    at mu (react-dom.production.min.js:257)
    at t.unstable_runWithPriority (scheduler.production.min.js:19)
    at qi (react-dom.production.min.js:122)
    at vu (react-dom.production.min.js:257)
    at react-dom.production.min.js:256
    at B (scheduler.production.min.js:17)
    at MessagePort.x.port1.onmessage (scheduler.production.min.js:14)
```

Screenshot from 2020-10-16 09-35-24

### PR DESCRIPTION
**Pull request by victorgcramos**
Due to some mistakes, the invoice details was not correctly using the updated fetch machine. This diff closes #2162 and fixes this issue

### Solution description

The solution was to use the fetch machine correctly, and create a simple fetching flow for invoices, just like we do for RFP proposals, but instead, we need to fetch the info for invoices proposals tokens.

### CODE REVIEW

File: src/containers/Invoice/helpers.js [hunk 1]
```diff
@@ -93,8 +93,10 @@ export const getInvoiceTotalHours = (invoice) => {
 };
 
 export const getInvoiceTotalExpenses = (invoice) => {
+  console.log(invoice);
   if (!invoice) return 0;
   return invoice.input.lineitems.reduce((total, item) => {
+    console.log("item", item, total);
```
Comment (tiagoalvesdulce) [inline src/containers/Invoice/helpers.js:99]: log

File: src/containers/Invoice/helpers.js [hunk 2]
```diff
@@ -93,8 +93,10 @@ export const getInvoiceTotalHours = (invoice) => {
 };
 
 export const getInvoiceTotalExpenses = (invoice) => {
+  console.log(invoice);
```
Comment (tiagoalvesdulce) [inline src/containers/Invoice/helpers.js:96]: log

Comment (victorgcramos) [reply src/containers/Invoice/helpers.js:96]: damn

**Review by tiagoalvesdulce — APPROVED** [review]
lgtm

**Pull request merged**